integration_tests/waypoints.cpp

```diff
@@ -64,276 +37,171 @@ TEST_F(SitlTest, MissionAddWaypointsAndFly)
     Device &device = dc.device();
 
     while (!device.telemetry().health_all_ok()) {
-        std::cout << "waiting for device to be ready" << std::endl;
+        LogInfo() << "waiting for device to be ready";
```

Comment:
- Capitalise sentence (prep for the example). 
- Do we need to think about case where health is never OK?

Reply:
Done, and yes we should add a timeout ad give up.